### PR TITLE
fix object interface issues

### PR DESCRIPTION
**Pull request by boellner**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -412,7 +420,7 @@ class openknx extends utils.Adapter {
             const out = outpath + id.replace(inpath, "");
             this.setState(out, {
                 val: state.val,
-                ack: true,
+                ack: false,
```
Comment (TA2k) [inline main.js:423]: ioBroker ACK false means this will interpret as user input and will send as write to the KNX bus

File: main.js [hunk 2]
```diff
@@ -236,30 +240,34 @@ class openknx extends utils.Adapter {
...
             knxVal = new Date(state.val);
         } else if (this.gaList.getDataById(id).native.valuetype == "composite") {
-            try {
-                knxVal = JSON.parse(state.val);
-            } catch (e) {
-                this.log.warn("stateChange: unsupported value format " + state.val + " for " + ga);
-                return;
-            }
+            //input from IOB is either object or string in object notation, type of this conversion is object
+            if (typeof state.val == 'object') {
```
Comment (TA2k) [inline main.js:244]: we should also allow the knx.0 format
1,0
and convert it to 
{priority: 0, data: 1}

File: main.js [hunk 3]
```diff
@@ -236,30 +240,34 @@ class openknx extends utils.Adapter {
...
+                knxVal = state.val;
+            } else
+                try {
```
Comment (TA2k) [inline main.js:247]: An improvement would be to check depending on the DPT if the object has the correct  properties

Comment (boellner) [reply main.js:247]: the underlying library also does some checks

**Review by TA2k — APPROVED** [review]

**Pull request merged**